packages/js/product-editor/src/components/iframe-editor/iframe-editor.tsx

```diff
@@ -264,11 +257,9 @@ export function IframeEditor( {
 								 bounds. */ }
 							<div className="woocommerce-iframe-editor__content-inserter-clipper" />
 						</BlockTools>
-						{ isRightSidebarOpen && (
-							<ComplementaryArea.Slot
-								scope={ SIDEBAR_COMPLEMENTARY_AREA_SCOPE }
-							/>
-						) }
+						<ComplementaryArea.Slot
+							scope={ SIDEBAR_COMPLEMENTARY_AREA_SCOPE }
+						/>
```

Comment:
This check caused some weird behaviour where the sidebar would not work anymore, it appeared that the `isRightSidebarOpen` would be stuck in an `undefined` or `null` state.
Given this is a `Slot` area anyway I don't see the need for us having a conditional around it, as the Fill logic should handle this.
Removing this fixed the issue and everything still seems to work as expected.